.github/workflows/playwright.yaml

```diff
@@ -1,22 +1,21 @@
 name: Playwright Tests
-on:
-  deployment_status:
+on: [pull_request]
+
 jobs:
-  test:
+  playwright:
     timeout-minutes: 60
     runs-on: ubuntu-latest
```

Comment:
```
jobs:
  playwright:
    defaults:
      run:
        working-directory: frontend
...
```

We can use `defaults` to set default working directory for entire job so that we don't have to use multiple `cd frontend`

Reply:
@palashgdev this is supposed to be merged with the existing `playwright` workflow job below.